2024/YoctoProjectDeveloperDay/practical-sec-tee.md

```diff
@@ -0,0 +1,1228 @@
+  + Can be implemented in TEE thus offering better security and extended functionality
+---
+
+# Secure Storage vs fTPM - Shameless Plug
+
+.center[
+
+### For those interested more about fTPM's
+
+[<img src="/img/FOSDEM_ftpm_ta_tee_blured.png" height="350px">
+](https://fosdem.org/2024/schedule/event/fosdem-2024-3097-securing-embedded-systems-with-ftpm-implemented-as-trusted-application-in-tee/)
+]
+
+---
+
+# TEE implementations
+
+.pure-table[
+| Company              | Product         | Hardware Used          | API Standard                  | Is Open-Source?       | Supported by Yocto? |
+|----------------------|-----------------|------------------------|-------------------------------|-----------------------|---------------------|
+| Alibaba              | Cloud Link TEE  | ?                      | GlobalPlatform                | ❌                    | ❌                  |
+| Apple                | Secure Enclave  | Separate processor     | Proprietary                   | ❌                    | ❌                  |
+| BeanPod              | ISEE            | ARM TrustZone          | GlobalPlatform                | ❌                    | ❌                  |
+| Huawei               | iTrustee        | ARM TrustZone          | GlobalPlatform                | ❌                    | ❌                  |
+| Google               | Trusty          | ARM / Intel            | Proprietary                   | Partially Open-Source | ❌                  |
+| Linaro               | OP-TEE          | ARM TrustZone          | GlobalPlatform                | ✔️                     | ✔️                   |
+| ProvenRun            | ProvenCore      | ARM TrustZone          | ?                             | ❌                    | ❌                  |
+| Qualcomm             | QTEE            | ARM TrustZone          | GlobalPlatform + Proprietary  | ❌                    | ❌                  |
+| Samsung              | TEEgris         | ARM TrustZone          | GlobalPlatform                | ❌                    | ❌                  |
+| TrustKernel          | T6              | Arm / Intel            | GlobalPlatform                | ? *                   | ❌                  |
+| Trustonic            | Kinibi          | ARM TrustZone          | GlobalPlatform                | ❌                    | ❌                  |
+| Open-TEE             | Open-TEE        | Emulation only         | GlobalPlatform                | ✔️                     | -                   |
+]
+
+.footnote[
+
+Sources:
+
+[wikipedia.org/Trusted_execution_environment](https://en.wikipedia.org/wiki/Trusted_execution_environment)
+
+]
+
+???
+
+- Wikipedia also specifies a formally-validated static partitioning über eXtensible
+Micro-Hypervisor Framework.
+    - Segway into Crosscon HV
+
+---
+
+<br>
+<br>
+<br>
+
+## .center[How to implement TEE in practice?]
```

Comment:
Please change to `How to integrate and use TEE`. I am not implementing TEE, there is **a huge** difference between `implementing` and `integrating`.